libs/tools/sharepoint/state.js

```diff
@@ -0,0 +1,45 @@
+import { signal } from '../../deps/htm-preact.js';
+
+const CONFIG = '/.milo/config.json';
```

Comment:
What config is this?

Reply:
This is a new config file which has been agreed upon for milo projects. 
cc: @auniverseaway

Reply:
Thanks, I just found it myself 👍